frontend/src/app/components/op-context-menu/op-context-menu.html

```diff
@@ -5,8 +5,8 @@
       role="menu"
       [ngClass]="{'-empty': items.length === 0 }">
     <ng-container *ngFor="let item of items">
-      <li *ngIf="item.divider && !item.hidden" class="dropdown-divider"></li>
-      <li *ngIf="!item.divider && !item.hidden">
+      <li *ngIf="item.divider" class="dropdown-divider"></li>
```

Comment:
This is a component that gets used in many places I believe. Are you sure that removing `&& !item.hidden` does not have side effects elsewhere?

Reply:
I think so. What I do in the component is to filter the items to those that are not hidden:
`this.items = this.locals.items.filter(item => !item?.hidden);` what it's basically the same that is done in the view.
I did that to get the '-empty' ngClass applied properly. It was not applied when there was only one item, but hidden, so the menu was still being opened even without visible items.